src/utils/appData.ts

```diff
@@ -18,7 +18,13 @@ async function getValidator(version: string): Promise<{ ajv: Ajv; validate: Vali
   validate = ajv.getSchema(version)
 
   if (!validate) {
-    const appDataSchema = await import(`@cowprotocol/app-data/schemas/v${version}.json`)
+    let appDataSchema
+
+    try {
+      appDataSchema = await import(`@cowprotocol/app-data/schemas/v${version}.json`)
+    } catch (e) {
+      throw new CowError(`AppData version ${version} does not exist`, 'MISSING_APP_DATA_VERSION')
+    }
```

Comment:
FYI, this has been added to app-data schema repo and will be replaced in a follow up PR https://github.com/cowprotocol/app-data/releases/tag/v0.0.1-RC.2